sdk/core/azure-core-amqp/inc/azure/core/amqp/models/amqp_properties.hpp

```diff
@@ -12,6 +12,14 @@
 #include <vector>
 
 struct PROPERTIES_INSTANCE_TAG;
+#if defined(__GNUC__) && (__GNUC__ > 4 || (__GNUC__ == 4 && __GNUC_MINOR__ >= 6))
+#pragma GCC diagnostic push
+#elif defined(__clang__) // !__clang__
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdocumentation-unknown-command"
+#endif // _MSC_VER
+
+///  @cond HIDDEN
```

Comment:
The template that gets specialized is from `_internal` namespace, yet Doxygen still loses its mind?

Reply:
doxygen doesn't get upset, clang gets upset. So I need to suppress the clang warning.